src/codemagic/tools/keychain.py

```diff
@@ -77,6 +77,32 @@ class KeychainArgument(cli.Argument):
         description='Encrypted p12 certificate password',
         argparse_kwargs={'required': False, 'default': ''},
     )
+    ALLOWED_APPLICATIONS = cli.ArgumentProperties(
+        flags=('-a', '--allow-app'),
+        key='allowed_applications',
+        description='Specify an application which may access the imported key without warning',
```

Comment:
i'd do something like `Specify an application which may access the imported key without warning with its absolute path or with its executable name if it's in the Path.

Reply:
Defaults are now `codesign` and `productsign` which are resolved to full paths later during action invocation. So it should eat really anything and further description becomes redundant.